src/sagemaker/estimator.py

```diff
@@ -37,6 +37,8 @@
     TRAINING_JOB_ROLE_ARN_PATH,
     TRAINING_JOB_ENABLE_NETWORK_ISOLATION_PATH,
     TRAINING_JOB_ENVIRONMENT_PATH,
+    TRAINING_JOB_PROFILE_CONFIG_PATH,
+    TRAINING_JOB_PROFILE_CONFIG_DISABLE_PROFILER_PATH,
```

Comment:
Can we just name this `TRAINING_JOB_DISABLE_PROFILER_PATH` as per the existing naming convention followed?